js/layout-javascript/small-card-code.js

```diff
@@ -324,19 +324,19 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
 
-      var $el = $(event.target);
+      var _that = $(this);
```

Comment:
Is `event.target` not the same as `this`?

Reply:
@tonytlwu no. When I changed this to $ el, an error appears and the list items do not open, everything works like this

Reply:
```suggestion
      var $el = $(this);
```

Then this should be all you need to change.

Reply:
@tonytlwu fixed